app/frontend/src/features/search/LocationAutocomplete/LocationAutocomplete.tsx

```diff
@@ -90,6 +97,15 @@ export default function LocationAutocomplete({
           searchSubmit(controller.field.value, "create-option");
         }
       }}
+      endAdornment={
```

Comment:
It'd be good to add a test to check clicking the button submits just like pressing enter